gapic/api/artman_speech_v1.yaml

```diff
@@ -0,0 +1,34 @@
+common:
```

Comment:
Since there's no longer a service config for `v1beta1`, can we go ahead and also remove the artman/GAPIC configs?